litestar/dto/_backend.py

```diff
@@ -781,6 +748,9 @@ def _create_msgspec_field(field_definition: TransferDTOFieldDefinition) -> Any:
     elif field_definition.default_factory is not None:
         kwargs["default_factory"] = field_definition.default_factory
 
+    if field_definition.name != field_definition.serialization_name:
+        kwargs["name"] = field_definition.serialization_name
+
```

Comment:
we could go a bit further than this and use msgspec's model-level renaming strategies when possible and only use the serialization name override for explicit field renames. This was just least disruptive to current code as a PoC, blah blah blah.

Reply:
done this - and it means we get the `kebab` strategy included for free as msgspec already support it